### ISSUE TITLE
Problem with cURL connection

### ISSUE THREAD

**Issue by jgm1986** [mami-project/KeyServer#14]
The user gets an error when launch cURL as KeyServer client:

```
curl: (35) gnutls_handshake() failed: The TLS connection was non-properly terminated.
```

cURL version details:

```
curl 7.47.0 (x86_64-pc-linux-gnu) libcurl/7.47.0 GnuTLS/3.4.10 zlib/1.2.8 libidn/1.32 librtmp/2.3
Protocols: dict file ftp ftps gopher http https imap imaps ldap ldaps pop3 pop3s rtmp rtsp smb smbs smtp smtps telnet tftp
Features: AsynchDNS IDN IPv6 Largefile GSS-API Kerberos SPNEGO NTLM NTLM_WB SSL libz TLS-SRP UnixSockets
```

**Comment by jgm1986** [comment]
Since Ubuntu v16.xx, cURL uses "GnuTLS" instead "OpenSSL" by default. According to the ["Compare SSL libraries"](https://curl.haxx.se/docs/ssl-compared.html) page, GnuTLS does not support SSLv2. KeyServer uses this version, so this is the origin of this problem.

**Comment by jgm1986** [comment]
As temporal solution, we have made a bash script for Ubuntu 16.04 for automatic cURL installation using OpenSSL instead GnuTLS. The KeyServer HTTPS side will be upgraded in future versions.

**Comment by jgm1986** [comment]
Problem solved in KeyServer v0.4.0.
